### ISSUE TITLE
Erro no checkout OSC-Magento-Brasil-6-Pro com módulo dos correios

### ISSUE THREAD

**Issue by marcosrogero** [deivisonarthur/OSC-Magento-Brasil-6-Pro#298]
Erro no checkout OSC-Magento-Brasil-6-Pro com módulo dos correios
: Erro inesperado. Descrição: Object reference not set to an instance of an object.
Segue um print da tela. Não sei porque isso ocorre e só aparece o método de pagamento corretamente quando a pessoa digita o CEP.
<img width="1257" alt="captura de tela 2017-05-22 as 15 40 34" src="https://cloud.githubusercontent.com/assets/28873388/26323386/0e28301c-3f05-11e7-8272-01e4a22fe9b8.png">

**Comment by Moi250** [comment]
Olá boa noite. 

Estou com mesmo problema, assim que instalei o modulo OSC-Magento-Brasil-6-Pro ele estava funcionando, apos eu configurar alguns campos como bairro, entre outros, e depois isso parou de funcionar o modulo do correio, agora estou apenas com o do Mercado Envio. 

Você já conseguiu resolver o seu?

**Comment by marcosrogero** [comment]
Não, inclusive eu desisti de mexer com Magento por causa disso. A plataforma é muito instável, está funcionando perfeitamente e de repente para é você não consegue fazer voltar a funcionar. Perdi um cliente grande e desisti de mexer com ecommerce por conta disso.

**Comment by PauloBoaventura** [comment]
Olá... 
Então essa implementação esta desatualizada e como nos encontramos sem soluções eficazes...
Estou em busca de soluções... 

Posto assim que tiver alguma... 
Abs!

**Comment by demiviola** [comment]
Olá, conseguimos arrumar este problema no módulo.
